storage/StorageExampleSwift/DownloadViewController.swift

```diff
@@ -37,16 +38,16 @@ class DownloadViewController: UIViewController {
     }
 
     // [START downloadimage]
-    storageRef.child(storagePath).write(toFile: fileURL, completion: { (url, error) in
-      if let error = error {
+    storageRef.child(storagePath).write(toFile: fileURL) { result in
+      switch result {
+      case let .success(url):
+        self.statusTextView.text = "Download Succeeded!"
+        self.imageView.image = UIImage(contentsOfFile: url.path)
+      case let .failure(error):
         print("Error downloading:\(error)")
         self.statusTextView.text = "Download Failed"
-        return
-      } else if let imagePath = url?.path {
-        self.statusTextView.text = "Download Succeeded!"
-        self.imageView.image = UIImage(contentsOfFile: imagePath)
       }
-    })
+    }
```

Comment:
This updates a doc snippet, which will generate an includecode bug if the snippet is linked in any docs. You should assign the includecode bug to me once it's generated and I'll update the doc accordingly.